src/main.cpp

```diff
@@ -64,6 +66,12 @@ int main(int argc, char **argv){
     lrf.setNumBeams(1000);
     lrf.setRangeLimits(0.01, 10);
 
+    // The bumper class implementation is a specialization of the lrf class
+    Bumper bumper;
+    bumper.setAngleLimits(-M_PI,M_PI);
+    bumper.setNumBeams(50);                 
+    bumper.setRangeLimits(0.01,1); 
```

Comment:
Ik zou een custom constructor voor je bumper maken waarin je de laser parameters zet die altijd hetzelfde zijn. Zodat je hier alleen nog maar het formaat van je robot hoeft in te geven